Dockerfile

```diff
@@ -23,7 +23,9 @@ RUN echo "usb 0x4b8 0x151" >> /etc/sane.d/epson2.conf
 COPY requirements.txt /tmp/requirements.txt
 RUN pip install -r /tmp/requirements.txt
 
-COPY data/ /tmp/data/
+ENV DEFAULT_CONFIG=/tmp/data/config/
+
+COPY data/config/ ${DEFAULT_CONFIG}
```

Comment:
this is so that the `setup.py` preforms the copy to the expected place afterwards right? But what's the point of having an environment variable?

Reply:
the point is that this environmental variable is used in the paths.py module.

Reply:
I don't think it should be fixed this way. The `setup.py` imports `setup_tools.py` script and runs `install_data_files()` with the intent of copying these files into the correct location if they are missing... something must be off there rather than solving it this way. However I believe there is a need to copy the entire `data/` into `/tmp/data`

Reply:
ok, now I see the problem. install_data_files() seems to do what it is supposed to do (copy stuff to /root/.scan-o-matic). But in the docker-compose file we map a local directory at this directory, so what is there from the beginning get lost, will try to fix this in some way.

Reply:
now solving this issue by modifying the entry point in the docker file so it copies required config files if missing after directories are mounted by docker-compose

Reply:
your new settings copy thing looks nicer than my old, but perhaps remove the old one completely and cover its entire functionality (some sub directories with files) rather than have code duplication.